modules/cli/src/main/scala/scala/cli/commands/publish/PublishLocalOptions.scala

```diff
@@ -3,16 +3,20 @@ package scala.cli.commands.publish
 import caseapp.*
 
 import scala.cli.commands.pgp.PgpScalaSigningOptions
-import scala.cli.commands.shared.{
-  CrossOptions,
-  HasSharedOptions,
-  MainClassOptions,
-  SharedOptions,
-  SharedPythonOptions,
-  SharedWatchOptions
-}
+import scala.cli.commands.shared._
 
 // format: off
+@HelpMessage(
+  s"""Publishes build artifacts to the local Ivy2 repository.
+     |
+     |The local Ivy2 repository usually lives under `~/.ivy2/local`.
+     |It is taken into account most of the time by most Scala tools when fetching artifacts.
```

Comment:
```suggestion
     |It is taken into account by Scala tools when fetching artifacts.
```
Are there tools that don't fetch dependencies from local ivy?

Reply:
I mean... Maven doesn't, I suppose.

Anyhow, it's a direct quote from our [publish local doc](https://scala-cli.virtuslab.org/docs/commands/publishing/publish-local)